docs/snippets/ov_hetero.cpp

```diff
@@ -47,8 +47,8 @@ auto compiled_model = core.compile_model(model, device);
         ov::device::priorities("GPU", "CPU"),
         // profiling is enabled only for CPU
         ov::device::properties("CPU", ov::enable_profiling(true)),
-        // FP16 inference precision only for GPU
-        ov::device::properties("GPU", ov::hint::inference_precision(ov::element::f16))
+        // FP32 inference precision only for CPU
+        ov::device::properties("CPU", ov::hint::inference_precision(ov::element::f32))
```

Comment:
1. We need to demonstrate different options for different devices, so let's change `enable_profiling` setting to GPU.
2. Let's update the description https://github.com/openvinotoolkit/openvino/blame/master/docs/OV_Runtime_UG/hetero_execution.md#L106 since we change devices for properties.